14_functions_lab/02.py

```diff
@@ -1,7 +1,37 @@
-"""
-Write a function that takes two arguments: 
-    A string
-    And a number
-If wrong types were passed in, raise an exception
-"""
-
+"""
+function that gets two paramets and check validity. (str and number)
+"""
+from types import *
+
+
+
+def func(*args):
+    parameters = { 'integers' : 0, 'strings' : 0 }

```

Comment:
Nice, but that wasn't the question... The task was to enforce first parameter to be an int and second to be a string